.travis.yml

```diff
@@ -0,0 +1,14 @@
+language: node_js
+node_js:
+  - 0.12
```

Comment:
can you bump this node version up to whatever version we are using, and update the readme